File: app/src/main/java/com/arny/pik/models/GenPlan.java

package com.arny.pik.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class GenPlan {

    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("data")
    @Expose
    private Data data;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "data:" + data;
    }
}

File: app/src/main/java/com/arny/pik/data/models/GenPlan.kt
package com.arny.pik.data.models

import com.arny.pik.utils.Utility
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class GenPlan {
    @Expose
    var title: String? = null
    @SerializedName("data")
    @Expose
    var data: Data? = null
    override fun toString(): String {
        return Utility.getFields(this)
    }
}
